adversarial_workflow/cli.py

```
@@ -2824,6 +2824,47 @@ def split(task_file: str, strategy: str = "sections", max_lines: int = 500, dry_
+        print(f"  {name:14} {config.description}")
+
+    print()
+
+    # Print local evaluators
+    local_evaluators = discover_local_evaluators()
+    if local_evaluators:
+        print(f"{BOLD}Local Evaluators{RESET} (.adversarial/evaluators/):")
+
+        # Group by primary name (skip aliases)
+        seen_configs = set()
+        for name, config in sorted(local_evaluators.items()):
```

Ruff (0.14.13)

Loop control variable `name` not used within loop body

(B007)